students/uss_tomasz/lesson_08_regular_expressions/8_2/strong_password_detection.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+
+def is_password_strong(password):
+    length_regex = re.compile(r'[a-zA-Z0-9]{8,}')
```

Comment:
Regex engine caches compiled patterns, so subsequent compilation of the same pattern will be much faster, but there is no need for that.
Move `length_regex = re.compile(r'[a-zA-Z0-9]{8,}')` out of this function and rename it, it's a constant, right?
same cases below